language/el.json

```diff
@@ -0,0 +1,60 @@
+{
+  "semantics": [
+    {
+      "label": "Τίτλος",
+      "placeholder": "Εδώ μπορείτε να προσθέσετε έναν τίτλο (προαιρετικό).",
+      "description": "Ο τίτλος που θα θέλατε να εμφανίζεται πάνω από την εικόνα"
+    },
+    {
+      "label": "Στοιχεία",
+      "entity": "στοιχειου",
+      "widgets": [
+        {
+          "label": "Βασικό"
+        }
+      ],
+      "field": {
+        "label": "Στοιχείο",
+        "fields": [
+          {
+            "label": "Εικόνα"
+          },
+          {
+            "label": "Ετικέτα",
+            "description": "Προαιρετική ετικέτα σημείου επιλογής. Βεβαιωθείτε ότι δεν είναι εκτενής ειδάλλως δεν θα εμφανίζεται."
+          },
+          {
+            "label": "Περιγραφή",
+            "placeholder": "Η περιγραφή της εικόνας μου...",
+            "description": "Περιγραφή της εικόνας (προαιρετικό)"
+          }
+        ]
+      }
+    },
+    {
```

Comment:
If you [compare the following section with the language template](https://github.com/otacke/h5p-agamotto/blob/master/language/.en.json), you will notice that your changes don't match.

I first thought you might have used an old version, but [your patch-1 branch was forked from the latest master version](https://github.com/otacke/h5p-agamotto/network).

Please make sure that your translations match the language template.

Reply:
...you're absolutely right. I 'll edit the file